ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_dev/ExplicitDisclosureIT.scala

```diff
@@ -745,13 +793,41 @@ object ExplicitDisclosureIT {
     )
   }
 
-  private def createEventToDisclosedContract(ev: CreatedEvent): DisclosedContract =
+  private def filterTxBy(
+      owner: binding.Primitive.Party,
+      template: Boolean = true,
+      interface: Boolean = false,
```

Comment:
I would like the blob inclusion to be called out explicitly on the argument list and then the test to cover both cases for interfaces i.e. with and without the blob

Reply:
I have pushed a fix with explicit filter. We extract blob though only if Record has not been delivered though, do you also want to change that?